guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to enable custom repositories by default on content hosts with a specific operating system version or architecture.
```

Comment:
Custom repositories are disabled by default now, so this should be reworded.

```suggestion
You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
```

(Side note: this is also being changed in the web UI: https://bugzilla.redhat.com/show_bug.cgi?id=2222979)

Reply:
(another side note: We've been moving away from saying "content hosts" in favor of just "hosts."